sleap/prefs.py

```diff
@@ -15,6 +15,7 @@ class Preferences(object):
         "medium step size": 10,
         "large step size": 100,
         "color predicted": False,
+        "color_by_instance_group": False,
```

Comment:
We do not want a boolean here. We want one of the four `str` options for which to distinctly color instances by:
https://github.com/talmolab/sleap/blob/b55d4ef914d9bffa68ef5be8fd22cb72041270ad/sleap/gui/app.py#L658

```suggestion
        "distinctly_color": "instances",
```